### PR TITLE
Slightly refactor stuff. Use more consistent code style.

### PR DESCRIPTION
**Pull request by paulmillr**

### CODE REVIEW

File: chrome/jsterm.js [hunk 1]
```diff
@@ -15,7 +15,15 @@ Cu.import("resource:///modules/WebConsoleUtils.jsm");
  * . ctrl-r
  */
 
-const JSTERM_MARK = "orion.annotation.jstermobject";
+let JSTERM_MARK = "orion.annotation.jstermobject";
+
```
Comment (paulrouget) [inline chrome/jsterm.js:19]: Why?

File: chrome/jsterm.js [hunk 2]
```diff
@@ -15,7 +15,15 @@ Cu.import("resource:///modules/WebConsoleUtils.jsm");
...
+let keys = {
+  enter: 13,
+  up: 38,
+  down: 40,
+  d: 68,
+  l: 76
+};
```
Comment (paulrouget) [inline chrome/jsterm.js:27]: Use this: http://mxr.mozilla.org/mozilla-central/source/dom/interfaces/events/nsIDOMKeyEvent.idl#8 instead

File: chrome/jsterm.js [hunk 3]
```diff
@@ -482,19 +491,15 @@ let JSTermUI = {
...
   this.buildDictionnary();
-
   this.sb = sandbox;
 }
 
 JSCompletion.prototype = {
   buildDictionnary: function() {
-    let JSKeywords = "break delete case do catch else class export continue finally const for debugger function default if import this in throw instanceof try let typeof new var return void super while switch with";
+    let JSKeywords = "break delete case do catch else class export continue finally let for debugger function default if import this in throw instanceof try let typeof new var return void super while switch with";
     this.dictionnary = JSKeywords.split(" ");
```
Comment (paulrouget) [inline chrome/jsterm.js:503]: let twice? Please keep const.

File: chrome/jsterm.js [hunk 4]
```diff
@@ -978,15 +1007,19 @@ JSTermLocalHistory.prototype = {
       this.cursor = null;
     }
   },
+
```
Comment (paulrouget) [inline chrome/jsterm.js:1010]: We can't change the above code as it is directly extracted from Firefox (just a couple of changes).

File: chrome/jsterm.js (hunk 1 above)
Comment (paulmillr) [inline chrome/jsterm.js:19]: sorry, I accidentally replaced consts with lets

**Pull request closed without merging**